<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlsDialog</class>
 <widget class="QDialog" name="ControlsDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>344</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Controls</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Moves</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>321</width>
         <height>291</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="horizontalSpacing">
         <number>50</number>
        </property>
        <item row="0" column="1">
         <widget class="QKeySequenceEdit" name="moveLeftKeySequenceEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="moveRightLabel">
          <property name="text">
           <string>Move right</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QKeySequenceEdit" name="moveRightKeySequenceEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="moveUpLabel">
          <property name="text">
           <string>Move up</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QKeySequenceEdit" name="moveUpKeySequenceEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="moveNextLeftLabel">
          <property name="text">
           <string>Move to next left</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QKeySequenceEdit" name="moveNextDownKeySequenceEdit"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="moveNextUpLabel">
          <property name="text">
           <string>Move to next up</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="moveDownLabel">
          <property name="text">
           <string>Move down</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QKeySequenceEdit" name="moveNextUpKeySequenceEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="moveNextRightLabel">
          <property name="text">
           <string>Move to next right</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QKeySequenceEdit" name="moveNextRightKeySequenceEdit"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="moveNextDownLabel">
          <property name="text">
           <string>Move to next down</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QKeySequenceEdit" name="moveNextLeftKeySequenceEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="QKeySequenceEdit" name="moveDownKeySequenceEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="moveLeftLabel">
          <property name="text">
           <string>Move left</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Undo/Redo</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>321</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="horizontalSpacing">
         <number>118</number>
        </property>
        <item row="0" column="1">
         <widget class="QKeySequenceEdit" name="undoKeySequenceEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="redoLabel">
          <property name="text">
           <string>Redo</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QKeySequenceEdit" name="redoKeySequenceEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="undoLabel">
          <property name="text">
           <string>Undo</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ControlsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ControlsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
